lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -16,263 +16,62 @@
 module: vmware_guest
 short_description: Manages virtual machines in vCenter
 description:
-- Create new virtual machines from templates or other virtual machines.
-- Manage power state of virtual machine such as power on, power off, suspend, shutdown, reboot, restart etc.,.
-- Modify, rename or remove a virtual machine.
+- This module can be used to create new virtual machines from templates or other virtual machines.
```

Comment:
How about this style of list?
This module can be used to:
 - create new virtual machines from templates or other virtual machines.
 - manage power state of virtual machines . . . .
etc.
Removing the repeated phrase makes the list easier to scan.

Reply:
OK.